extensions/auth/opa/impl/build.gradle.kts

```diff
@@ -58,3 +59,64 @@ dependencies {
   testImplementation(project(":polaris-async-java"))
   testImplementation(project(":polaris-idgen-mocks"))
 }
+
+// Task to generate JSON Schema from model classes
+tasks.register<JavaExec>("generateOpaSchema") {
+  group = "documentation"
+  description = "Generates JSON Schema for OPA authorization input"
+  classpath = sourceSets["main"].runtimeClasspath
+  mainClass.set("org.apache.polaris.extension.auth.opa.model.OpaSchemaGenerator")
+  args("${projectDir}/opa-input-schema.json")
```

Comment:
Nit/tip: Using `projectDir` here is superfluous. The default working directory for the executed process is the project directory.